src/views/QuestionBank/js/question-bank-detail.js

```diff
@@ -87,6 +102,9 @@ export default {
       }
     },
     failFetchingQuestionBankQuestionList () {
+      this.isLoading = false
+      this.failLoadQuestion = true
+      //TODO : CHANGE TO EMPTY STATE
```

Comment:
already working this TODO for the next development?

Reply:
Already implemented, forgot to remove TODO